.github/workflows/update.yml

```diff
@@ -19,16 +19,22 @@ jobs:
           ref: main
       - name: Install Nix
         uses: cachix/install-nix-action@V27
+        with:
+          nix_path: nixpkgs=channel:nixpkgs-unstable
       - name: Setup Cachix
         uses: cachix/cachix-action@v15
         with:
           name: devenv
       - name: Install devenv
         run: nix-env -if https://install.devenv.sh/latest
```

Comment:
Hi, @sestrella! I'm wondering if it would be better to pin an specific version of devenv here to avoid the workflow breaks with an update?

Reply:
@fm7-1 sure! Great recommendation